src/Prefabs/Tools/Axe/Axe.et

```diff
@@ -6,17 +6,26 @@ GenericEntity : "{F3919A62ADD6123A}Prefabs/Tools/Tool_Base.et" {
     "{430CD8162D17C2FF}Sounds/Weapons/_SharedData/Melee/Weapons_Melee_Pistol.acp" "{8D040E18DFEC29AF}Sounds/Weapons/_SharedData/PickUp/Weapons_PickUp_HandGun.acp"
    }
   }
+  EL_PersistenceComponent "{5A4C7559CF69AAF0}" {
+   Enabled 1
+   m_pSaveData EL_ItemSaveData "{5A4C7559EABBF0B0}" {
+    m_aComponents {
+     EL_BaseInventoryStorageComponentSaveData "{5A4D31A91DD9886E}" {
+     }
+    }
+   }
+  }
```

Comment:
Does it need to be explicitly enabled? It should be by default.
Also, I do not think it needs any components besides transformation (that is currently missing), as an axe/tool does not store anything. So I would move this change into tool base anyway, as it will be likely the same basis for all tools.

Reply:
Deleted